jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java

```diff
@@ -13,6 +13,7 @@
  */
 package tech.catheu.jnotebook;
 
+import com.google.common.io.Files;
```

Comment:
nit: can't remember if it is an explicitly declared dependency
I'll check

Reply:
it was there before.